pvr.argustv/changelog.txt

```diff
@@ -1,3 +1,11 @@
+v19.1.2
```

Comment:
Sorry, I spoke too soon. As you are adding functionality the version number should `19.2.0`. You would use `19.1.2` if you had fixed a bug.